### PR TITLE
fix: Incorrect currency shown at /bsx

### LINKED ISSUE #3294: Basilisk Subscan link
Basilisk will have Subscan, rather just hide it for now and once we will run on official testnets it can be add again.

_Originally posted by @yangwao in https://github.com/kodadot/nft-gallery/issues/3280#issuecomment-1166582210_

// Once Basilisk becomes available on Subscan, let's add the functionality we've hidden in 
- #3280

### LINKED ISSUE #4039: Incorrect currency shown at /bsx
### What happened?

When you open a history table you can see the price in BSX instead of KSM

### Please reproduce in steps

1. Open any NFT with price
2. Toggle history table
3. the price is denominated in BSX instead of KSM

### Expected Behavior

There should be `KSM` instead of `BSX`

Please use the proper solution (we have component for that)

### What browsers are you seeing the problem on?

_No response_

### At which address did you encounter bug?

_No response_

### Are you logged in?

_No response_

### Which wallet you are using?

_No response_

### Screenshots

Screenshot 2022-09-28 at 10 23 00

### Relevant log output

_No response_

### Payment link for reward

https://kodadot.xyz/transfer/?target=%3CMy_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address%3E

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #4039, closes #3294
- [ ] Requires deployment <rubick/magick_issue>
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

bsx subscan link
https://basilisk.subscan.io/block/1879043

image

### CODE REVIEW

File: utils/config/chain.config.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ const chainPropertyMap: Config<ChainProperties> = {
   bsx: toChainProperty(
     10041,
     12,
-    'BSX',
+    'KSM',
     'https://calamar.play.hydration.cloud/basilisk/search?query='
```
Comment (petersopko) [inline utils/config/chain.config.ts:34]: just saw this link here, maybe we want to switch this up for https://basilisk.subscan.io/? 
ref:
- https://github.com/kodadot/nft-gallery/issues/3294

Comment (Jarsen136) [reply utils/config/chain.config.ts:34]: I would take a look

Comment (Jarsen136) [reply utils/config/chain.config.ts]: Done

**Comment by petersopko** [comment]
please, search for all "calamar" related links, one is in `supportedExplorers`
I've noticed two things:
1. it looks like subscan is showing only BSX token on Basilisk Mainnet, i.e. I can't see the Kusama values there yet
image

3. it looks like calamar dropped support for Basilisk anyways, so I guess subscan is good enough for now
image
image

**Comment by Jarsen136** [comment]
> supportedExplorers

It's used for the explorer link for address.
The bad news is that neither basilisk nor calamar can properly display address details : )

https://basilisk.subscan.io/account/bXk6PXHj9NnYkiZ45ETrq4iLiozW8a4khe8ZmqDkdbLJNNMaE
https://calamar.play.hydration.cloud/basilisk/search?query=bXk6PXHj9NnYkiZ45ETrq4iLiozW8a4khe8ZmqDkdbLJNNMaE

**Comment by petersopko** [comment]
> It's used for the explorer link for address. The bad news is that neither basilisk nor calamar can properly display address details : )

are you sure you have the right address there? it works for mine https://basilisk.subscan.io/account/bXkJuVse1JQXeCC7ooaGUqWKPgHGStuK5qNzZmmeVea2YDn84

**Comment by Jarsen136** [comment]
> > It's used for the explorer link for address. The bad news is that neither basilisk nor calamar can properly display address details : )
> 
> are you sure you have the right address there? it works for mine https://basilisk.subscan.io/account/bXkJuVse1JQXeCC7ooaGUqWKPgHGStuK5qNzZmmeVea2YDn84

My bad. 
Already fixed.

**Review by petersopko — APPROVED** [review]
wfm

**Review by kkukelka — APPROVED** [review]
code lgtm

**Comment by kkukelka** [comment]
pay 50

**Comment by kkukelka** [comment]
@yangwao is the bot broken?

**Comment by kkukelka** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 44.06 USD/KSM ~ 1.135 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0x8d60dfc8dc81085baecafbf0a88f98679e58fa5bcf714b3f54da78572661026c](https://kusama.subscan.io/extrinsic/0x8d60dfc8dc81085baecafbf0a88f98679e58fa5bcf714b3f54da78572661026c)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by kkukelka** [comment]
nvm haha

**Pull request merged**